its/src/test/java/com/sonar/maven/it/suite/JavaTest.java

```diff
@@ -120,6 +121,31 @@ void setJavaVersionProperties() throws IOException {
       entry("sonar.java.target", "1.8"));
   }
 
+  @Test
+  void whenPropertyJavaHomeExistsThenItIsUsedAsDefaultValue() throws IOException {
```

Comment:
We are on the right track with this test title but I see a couple of improvements we can make here
1. Specify for which property we get a default value. We should not have to dive into the code of the test to check what property is being set.
2. Rewrite the test title in snake_case. Our projects tend to use snake_case to make the test title more legible

Reply:
:+1: